libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -103,6 +103,10 @@
         PUSH_AUTHENTICATION_FAILED,
         PUSH_AUTHENTICATION_IGNORED,
         SETTINGS_LANGUAGE_SELECTION_FORCED,
+        NOTIFICATION_SETTINGS_LIST_OPENED,
+        NOTIFICATION_SETTINGS_STREAMS_OPENED,
```

Comment:
Where does these events are going to be used?
I'm even more curious about `NOTIFICATION_SETTINGS_STREAMS_OPENED` :)